visualization_tool/src/components/ResourcesList/ResourcesList.tsx

```diff
@@ -1,9 +1,45 @@
+import {Resource} from '../../types/resources';
+
 import './ResourcesList.css';
 
-const ResourcesList = () => {
+type ResourcesListProps = {resources: Resource[]};
+
+const ResourcesList = ({resources}: ResourcesListProps) => {
   return (
     <div className="resources-list">
-      <h1>Found Resources</h1>
+      <h1>{resources.length > 0 ? 'Found Resources' : 'No Resources Found'}</h1>
+      <div className="resources-list__table">
+        {resources
+          // sort resources by creationTimestamp
+          .sort((a, b) => {
+            return (
+              new Date(a.creationTimestamp).getTime() -
+              new Date(b.creationTimestamp).getTime()
+            );
+          })
+          .map(resource => {
+            return (
+              <div className="resources-list__table__card" key={resource.id}>
+                <img src={`./icons/resources/${resource.type}.png`} alt="" />
+                <p className="resource-name">{resource.name}</p>
+                <p className="resource-type">{resource.type}</p>
+                <p
+                  className="resource-status"
+                  style={
```

Comment:
Maybe extract this into a helper function?

Reply:
Which part exactly do you refer to

Reply:
The snippet of code that sets the style.

Reply:
Done